src/utils.ts

```diff
@@ -5,6 +5,11 @@ import type {
   JsonRpcSuccess,
 } from './JsonRpcEngine';
 
+export const hasProperty = (
+  object: Object, // eslint-disable-line @typescript-eslint/ban-types
```

Comment:
Should this be `any`?

Reply:
I don't think so, we only want to pass it to things that `hasOwnProperty` won't throw for. Ref: https://www.typescriptlang.org/docs/handbook/basic-types.html#object